test/channels.jl

```diff
@@ -127,7 +128,7 @@ using Distributed
 
     # channeled_tasks
     for T in [Any, Int]
-        chnls, tasks = Base.channeled_tasks(2, (c1,c2)->(@assert take!(c1)==1; put!(c2,2)); ctypes=[T,T], csizes=[N,N])
+        chnls, tasks = Base.channeled_tasks(2, (c1,c2)->(@assert take!(c1)==1; put!(c2,2); sleep(1.0)); ctypes=[T,T], csizes=[N,N])
```

Comment:
are the `sleep`'s here necessary?

Reply:
good question. If this sleep is necessary to get another task executed, why not use direct means like conditional variable (if it fits the test's paradigm)

Reply:
That's why I added the comment above this testset that these tests should be redesigned -- for some reason, all these tests rely on an understanding of how tasks are scheduled (or rather how tasks _were_ scheduled). With the new runtime, such assumptions no longer hold.